### ISSUE TITLE
docs(search): update to A11y search doc

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon-website#729]
Please update the A11y Search documentation as follows:

"The Search Carbon Component contains a collection of user interface controls, that combine to create search functionality to content on the website. Labels are used on each button so users of assistive technologies understand what the button is used for with helpful placeholder text within the search field."

**Comment by snidersd** [comment]
I just noticed that the live version of this content is also missing the Resources navigation link that should link down to the Resources section.

**Comment by laurenmrice** [comment]
Our Accessibility expert has created a new dedicated tab for Accessibility documentation for the Search component which resolves this issue. https://carbondesignsystem.com/components/search/accessibility . Going to close.
